src/main/java/io/pravega/connectors/hadoop/PravegaInputFormat.java

```diff
@@ -77,15 +92,24 @@ protected PravegaInputFormat(ClientFactory externalClientFactory) {
         final String deserializerClassName = Optional.ofNullable(conf.get(PravegaInputFormat.DESERIALIZER)).orElseThrow(() ->
                 new IOException("The event deserializer must be configured (" + PravegaInputFormat.DESERIALIZER + ")"));
 
+        final StreamCut startStreamCut = getStreamCutFromPositionsJson(
```

Comment:
My understanding here is that this will not be required once https://github.com/pravega/pravega/issues/2490 is resolved, right?

Reply:
Yes, that's correct